section/DrawFunc.cpp

```diff
@@ -116,8 +116,10 @@ namespace Moho
             return t;
         }
 
-        void ReleaseTexture(Texture *t) {
-            asm("call 0x004260B0;" : : [t] "a"(t) : "edx", "ecx");
+        void *ReleaseTexture(Texture *t) {
+            void *__result;
+            asm("call 0x004260B0;" : "=a"(__result) : [t] "a"(t) : "edx", "ecx");
```

Comment:
Function `0x004260B0` doesn't return anything

Reply:
Yes, but it alters `eax` register. I'm not sure how compiler may behave without return value, since it is different from argument.

Reply:
And you can't put it in clobbers eaither